src/coreclr/jit/gentree.cpp

```diff
@@ -17601,6 +17601,12 @@ GenTree* Compiler::gtGetSIMDZero(var_types simdType, CorInfoType simdBaseJitType
                     case CORINFO_TYPE_ULONG:
                         assert(simdHandle == m_simdHandleCache->Vector128ULongHandle);
                         break;
+                    case CORINFO_TYPE_NATIVEINT:
+                        assert(simdHandle == m_simdHandleCache->Vector128NIntHandle);
```

Comment:
Do we need to also handle `CORINFO_TYPE_NATIVEINT` and `CORINFO_TYPE_NATIVEUINT` for 64-bit vectors (`TYP_SIMD8`) in this function?

Reply:
No, it and `CORINFO_TYPE_LONG` and `CORINGO_TYPE_ULONG` get some special handling elsewhere in the JIT since they are the same size as `Vector64` itself.